metadata_checks/metadata_checker.py

```diff
@@ -0,0 +1,35 @@
+"""Module for metadata checks."""
+
+import nuke
+
+
+def assert_has_metadata(node: nuke.Node, metadata: dict[str, str]) -> None:
```

Comment:
I’d suggest to change the name to assert_contains_metadata() as I expected has metadata to be an equal check of metadata. Contains is more explicit that it is a subset of the metadata